src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -64,25 +85,25 @@
     <!-- Download url for the SonarJS plugin. Update this url to embed a new version of the plugin. -->
     <PluginUrl>https://binaries.sonarsource.com/Distribution/sonar-javascript-plugin/sonar-javascript-plugin-$(EmbeddedSonarJSAnalyzerVersion).jar</PluginUrl>
     <TypeScriptFolderName>ts</TypeScriptFolderName>
-    
+	
```

Comment:
Saving an old-style project in VS messes up the layout e.g. removes empty lines that were added to make the file easier to read.
I'll convert this file to be SDK-style later in this sprint/next sprint.